conda-recipe/build.sh

```diff
@@ -4,7 +4,6 @@
 # by DPC++ compiler conda packages. Will need to be added to DPC++ compiler
 # activation scripts.
 export LDFLAGS="$LDFLAGS -Wl,-rpath,$PREFIX/lib"
```

Comment:
Isn't that also relating to the W/A and should be removed? At least the above comment claims regarding that.